pages/index-new.js

```diff
@@ -117,7 +118,7 @@ const Page = () => {
         <Public>
           <MenuBar>
             <Link href="/about">About</Link>
```

Comment:
Non-blocker: want to strip this link while here @edderrd? Seems like we don’t need it at present.

Reply:
striped @mckelvey